### ISSUE TITLE
Food Recall warning: Text needs translated on French page 

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#101]
URL: https://hcdev.openplus.ca/fr/avis-rappel/test-rappel-certains-produits-vegetaliens-et-vegetariens-marque-happy-veggie-world
User Agent: Chrome
Language: French

Issue Description: There is English text on the French page. Please see screenshot. All text should contain the language of the page.

Steps to reproduce: 
1. Select the url listed above
2. Visually scan the page
3. 'Media and public inquiries' under 'Renseignements supplémentaires' needs translated
4. Click on the 'Détails' link
5. Please refer to highlighted areas in the provided screenshot for text that needs to be corrected

Expected results: All text should contain the correct language of the page. These should be translated. If not then the text should be wrapped within a lang attribute to ensure that the words are properly said by assistive technologies.
![image](https://user-images.githubusercontent.com/68117281/136611040-b7171e41-4428-49ea-9bb3-63aca1bdf200.png)

**Comment by echouin** [comment]
@mmiron-HC completed.

**Comment by ndolter** [comment]
Validated that the translations have been made
 https://hcdev.openplus.ca/fr/avis-rappel/test-rappel-certains-produits-vegetaliens-et-vegetariens-marque-happy-veggie-world
